test/unit/sql/select.test.js

```diff
@@ -1,14 +1,14 @@
 'use strict';
 
-const Support = require('../support'),
-  DataTypes = require('../../../lib/data-types'),
-  Model = require('../../../lib/model'),
-  util = require('util'),
-  chai = require('chai'),
-  expect = chai.expect,
-  expectsql = Support.expectsql,
-  current = Support.sequelize,
-  sql = current.dialect.QueryGenerator,
+const Support = require('../support'), 
```

Comment:
was this due to `eslint --fix`? just added one space at the end of line in some files, no other change

Reply:
probably.  For this reason it's good to strongly enforce formatting

Reply:
After all this conversion is done, I am planning to use prettier on our codebase

Reply:
Sounds good.  I recommend using `husky`/`lint-staged` to automatically format staged files, I could help set that up when the time comes

Reply:
I think this was actually caused by `recast` after my codemods, because it didn't happen to every file...and I think that `eslint --fix` doesn't remove the trailing whitespace

Reply:
okay I undid all these whitespace-only changes, for some reason it only happened in this directory